components/Cards/index.js

```diff
@@ -17,3 +17,46 @@
 // </div>
 //
 // Create a card for each of the articles and add the card to the DOM.
+
+const cardsContainer = document.querySelector('.cards-container');
+
+axios.get('https://lambda-times-backend.herokuapp.com/articles')
+    .then(response => {
+        console.log(response)
+        for(let article in response.data.articles) { 
```

Comment:
in going with the dry principle you could save response.data.articles  into a reusable variable